src/main/scala/it/pagopa/interop/tenantprocess/api/adapters/ApiAdapters.scala

```diff
@@ -104,7 +104,7 @@ object ApiAdapters {
               verificationDate = now,
               renewal = seed.renewal.toDependency,
               expirationDate = seed.expirationDate,
-              extensionDate = None
+              extensionDate = seed.expirationDate.orElse(None)
```

Comment:
this is equivalent to
```suggestion
              extensionDate = seed.expirationDate
```